src/Stripe/Services/CountrySpecs/StripeCountrySpecsService.cs

```diff
@@ -0,0 +1,45 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Threading.Tasks;
+
+namespace Stripe
+{
+    public class StripeCountrySpecsService : StripeService
+    {
+        public StripeCountrySpecsService(string apiKey = null) : base(apiKey) { }
+
+        public virtual StripeCountrySpec Get(string country, StripeRequestOptions requestOptions = null)
+        {
+            return Mapper<StripeCountrySpec>.MapFromJson(
+                Requestor.GetString(this.ApplyAllParameters(null, $"{Urls.CountrySpecs}/{country}", false),
+                SetupRequestOptions(requestOptions))
+            );
+        }
+
+        public virtual IEnumerable<StripeCountrySpec> List(StripeChargeListOptions listOptions = null, StripeRequestOptions requestOptions = null)
+        {
+            return Mapper<StripeCountrySpec>.MapCollectionFromJson(
+                Requestor.GetString(this.ApplyAllParameters(listOptions, $"{Urls.CountrySpecs}", true),
+                SetupRequestOptions(requestOptions))
+            );
+        }
+
+#if !PORTABLE
```

Comment:
you don't need to do this anymore - async is available in the PCL :+1:   (remove precompiler)